webclient/index.html

```diff
@@ -105,7 +105,7 @@
       background-color: #DEDEDE;
```

Comment:
This file is going to be replaced with a proper web-app soon enough. At this point modifications are being made just to test updated web-socket protocols